app/src/main/res/values-vi/strings.xml

```diff
@@ -0,0 +1,205 @@
+    <string name="message">Tin nhắn</string>
+    <string name="ok">OK</string>
+    <string name="cannot_load_the_data">Không thể tải dữ liệu</string>
+    <string name="email_is_Empty">Email không được để trống !</string>
+    <string name="password_is_Empty">Mật khẩu không được để trống!</string>
+    <string name="you_need_to_choose_a_domain_to_continue">Bạn cần chọn miền để tiếp tục!</string>
+    <string name="none">Không</string>
+    <string name="search">Tìm kiếm...</string>
+    <string name="product">Sản phẩm</string>
+    <string name="shop">Cửa hàng</string>
+    <string name="please_click_back_again_to_exit">Làm ơn bấm Back lần nữa để thoát!</string>
+    <string name="confirm_new_password_does_not_match">Xác nhận mật khẩu mới không trùng khớp</string>
+    <string name="current_password_is_empty">Mật khẩu hiện tại để trống</string>
+    <string name="update_profile_success">Cập nhật thông tin cá nhân thành công</string>
+    <string name="nothing_change">Không có gì thay đổi</string>
+    <string name="change_password">Thay đổi mật khẩu</string>
+    <string name="change_domain">Thay đổi miền</string>
+    <string name="shop_name">Tên cửa hàng: </string>
+    <string name="product_quantity">Số lượng sản phẩm: </string>
+    <string name="owner">Chủ cửa hàng: </string>
+    <string name="rating">Xếp hạng: </string>
+    <string name="comments">Bình luận</string>
+    <string name="products_shop">Sản phẩm trong shop</string>
+    <string name="see_all">Xem tất cả</string>
+    <string name="title_shop">Cửa hàng :\u0020 </string>
+    <string name="shop_information">Thông tin cửa hàng</string>
+    <string name="shop_owner_information">Thông tin chủ cửa hàng</string>
+    <string name="time">Thời gian:</string>
+    <string name="pending">Chờ</string>
+    <string name="accepted">Chấp nhận</string>
+    <string name="rejected">Từ chối</string>
+    <string name="completed">Hoàn thành</string>
+    <string name="step_one_check_order">Bước 1: Kiểm tra đơn hàng</string>
+    <string name="step_two_close_order">Bước 2: Đóng đơn hàng</string>
+    <string name="continue_check">Tiếp tục</string>
+    <string name="shop_related">Cửa hàng liên quan</string>
+    <string name="add_a_comment">Thêm bình luận</string>
+    <string name="quantity">Số lượng : %d</string>
+    <string name="prices">Giá : %s</string>
+    <string name="start_date">Ngày bắt đầu </string>
+    <string name="end_date">Ngày kết thúc </string>
+    <string name="filter">Lọc</string>
+    <string name="your_order_history">Lịch sử đặt hàng của bạn</string>
+    <string name="accept">Chấp nhận</string>
+    <string name="reject">Từ chối</string>
+    <string name="select_all">Chọn tất cả</string>
+    <string name="see_more">Xem thêm..</string>
+    <string name="button_join_domain">Nút tham gia vào miền</string>
+    <string name="image_shop_management">Ảnh quản lý cửa hàng</string>
+    <string name="domain_location">Vị trí miền</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.domain_location` appears to be unused
 Unused resources make applications larger and slow down builds.